index.html

```diff
@@ -15,6 +15,21 @@
 </head>
 
 <body>
+  <nav class="top-nav">
+    <h1>Fun Bus</h1>
```

Comment:
Elements inside the `<nav>`, semantically, should only be the nav menu (`<a>` tags). And taking out `<h1>` in the `<nav>` will give you more option how to style it. 😄